src/config.ts

```diff
@@ -8,27 +8,55 @@ export enum ConfigKeys {
     deckType = 'deckType',
     nextDraw = 'nextDraw',
     enableNotification = 'enableNotification',
-    pileUp = 'pileUp'
+    pileUp = 'pileUp',
+    difficultyLevel = 'difficultyLevel',
+    aggregatePile = 'aggregatePile',
+    useWeight = 'useWeight'
 }
 
 export class Config {
+    static getConfigValue(key: ConfigKeys): unknown {
+        return vscode.workspace.getConfiguration(CONFIG_NAME).get(key);
+    }
+
     static get pickEvery(): number {
-        return Number(vscode.workspace.getConfiguration(CONFIG_NAME).get(ConfigKeys.pickEvery));
+        return Number(Config.getConfigValue(ConfigKeys.pickEvery));
     }
 
     static get isBadgeNotificationEnabled(): boolean {
-        return Boolean(vscode.workspace.getConfiguration(CONFIG_NAME).get(ConfigKeys.enableBadgeNotification));
+        return Boolean(Config.getConfigValue(ConfigKeys.enableBadgeNotification));
     }
     
     static get isNotificationEnabled(): boolean {
-        return Boolean(vscode.workspace.getConfiguration(CONFIG_NAME).get(ConfigKeys.enableNotification));
+        return Boolean(Config.getConfigValue(ConfigKeys.enableNotification));
     }
 
     static get deckType(): string {
-        return String(vscode.workspace.getConfiguration(CONFIG_NAME).get(ConfigKeys.deckType));
+        return String(Config.getConfigValue(ConfigKeys.deckType));
     }
 
     static get pileUp(): boolean {
-        return Boolean(vscode.workspace.getConfiguration(CONFIG_NAME).get(ConfigKeys.pileUp));
+        return Boolean(Config.getConfigValue(ConfigKeys.pileUp));
+    }
+
+    static get useWeight(): boolean {
+        return Boolean(Config.getConfigValue(ConfigKeys.useWeight));
+    }
+
+    static get aggregatePile(): boolean {
```

Comment:
Can you update in `extension.ts`, inside the `onDidChangeConfiguration` callback to automatically enable pile up if you enable `aggregatePile` and to disable `aggregatePile` otherwise it does some weird shinanigan:
![image](https://user-images.githubusercontent.com/69776305/212490875-3eb7acc5-6795-4190-9f21-c70f407a6582.png)

Reply:
Fort mort!